lib/l10n/intl_it.arb

```
@@ -694,5 +694,64 @@
+  },
+  "notificationSendInvoice": "Si prega di inviare una fattura",
+  "notificationSendInvoiceAmount": "Si prega di inviare una fattura per {amount} satoshi",
+  "@notificationSendInvoiceAmount": {
+    "placeholders": {
+      "amount": {
+        "type": "String"
+      }
+    }
+  },
+  "notificationFiatSent": "Il pagamento fiat è stato inviato. In attesa di conferma.",
+  "notificationBitcoinReleased": "Bitcoin è stato rilasciato! La tua fattura dovrebbe essere pagata presto.",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)